<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuntingMainWindow</class>
 <widget class="QMainWindow" name="HuntingMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HuntingMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>120</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="newGameButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>102</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Új játék</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseContinueButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>102</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Megállít</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="pointCounter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="dimension3Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>102</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>3x3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dimension5Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>102</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5x5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dimension7Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>102</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>7x7</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="playerArea" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>631</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="stepping">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vadász lép</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="up">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="right">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="down">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF_jl">
    <property name="title">
     <string>Fájl</string>
    </property>
    <addaction name="saveButton"/>
    <addaction name="loadButton"/>
   </widget>
   <addaction name="menuF_jl"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveButton">
   <property name="text">
    <string>Mentés</string>
   </property>
  </action>
  <action name="loadButton">
   <property name="text">
    <string>Betöltés</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
